src/Video.tsx

```diff
@@ -118,11 +121,7 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
       [posterResizeMode],
     );
 
-    const src = useMemo(() => {
-      if (!source) {
```

Comment:
the is !source shall be kept I think (see my other comment)

Reply:
oh yeah it can be possible. Thanks.
it was my fault.
Fixed on this commit
https://github.com/react-native-video/react-native-video/pull/3487/commits/7944195e9dabcc6d4d9d6c7519903742ccb8df80